src/main/java/at/hannibal2/skyhanni/config/features/dungeon/DungeonConfig.java

```diff
@@ -56,6 +56,12 @@ public class DungeonConfig {
     @FeatureToggle
     public boolean highlightTeammates = true;
 
+    @Expose
+    @ConfigOption(name = "Architect Notifier", desc = "Notifies you to use the Architect in Dungeons when a puzzle is failed.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean architectNotifier = false;
```

Comment:
```suggestion
    public boolean architectNotifier = true;
```
I think this is fine to auto enable